#1 histogram modification

wernerd-econ opened the pull request:
Closes #1 

Per the instructions in the practice task 3, I have updated the code in the data folder so that the histogram appears in percentages instead of counts. This updated histogram also shows up in the slides. 

I have also taken the liberty to fix a few typos (and the outdated theme of the slides) that prevented the run_all.py from running smoothly. 

I am submitting this PR for @xingtong-jiang to review whenever is convenient for him. Thanks!

wernerd-econ committed:
#1 Percentage Histogram

xingtong-jiang commented:
Thanks @wernerd-econ! I would probably write the commit message as 
```
#1 Display percentages in histogram
```
Other than that, your work is perfect. Approving the PR now! Feel free to merge the issue branch to into the main branch when you have a moment.

wernerd-econ commented:
### Final Summary Comment
Per the instructions in Practice [Task 3](https://gentzkowlab.github.io/GentzkowLabTemplate/task3.html), I modified the code in the data folder of the repo so that the histogram in  `output/chips_sold.pdf` displays percentages instead of counts. 

I also fixed a few typos (and the outdated theme of the slides) that prevented the run_all.py from running smoothly.

This #2 PR was approved by @xingtong-jiang.

xingtong-jiang commented:
@wernerd-econ Thanks for the great work! A quick reminder that a better name for the PR commit would be:

```
PR for #<issue number>: <issue name>
```

Typically we don't need to mention "Squash Merge" in the commit message, since it's our default merge method.
